functions/model/tasks/lib.js

```diff
@@ -53,7 +53,7 @@ const { db } = require("../application/lib");
  * @param {any} prNumber
  * @return {any}
  */
-exports.setTask = function(orgDomain, taskId, title, des, priority, difficulty, creator, assignee, reporter, estimatedTime, status, project, loggedWorkTotalTime, workDone, sprintNumber, storyPointNumber, creationDate, completiondate, orgId, teamId, type, taskFileCounter, linkCounter = 0, lastUpdatedDate, watchers, prLink="", prApiLink="", prNumber=null) {
+exports.setTask = function(orgDomain, taskId, title, des, priority, difficulty, creator, assignee, reporter, estimatedTime, status, project, loggedWorkTotalTime, workDone, sprintNumber, storyPointNumber, creationDate, completiondate, orgId, teamId, type, taskFileCounter, linkCounter = 0, lastUpdatedDate, watchers, prLink="", prApiLink="", prNumber=null, milestoneId) {
```

Comment:
Better to put milestone Id before default variables, as this could break the code. Also check the use case of default arguments